code/modules/unit_tests/_unit_tests.dm

```diff
@@ -0,0 +1,5 @@
+//include unit test files in this module in this ifdef
+
+#ifdef UNIT_TESTS
+#include "unit_test.dm"
+#endif

```

Comment:
This should probably be in compile options

Reply:
Compile options should be flags only imo, also you lose access to all the defines if you do that.